src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -292,6 +306,7 @@ def main():
     root_dir = args.root_dir
 
     task_mgr = TFTaskManager(kbase_collection, load_ver, tool, source_data_dir, root_dir=root_dir)
+    task_mgr.check_preconditions(args.force)
```

Comment:
If the preconditions check needs to happen every time, just call it from the class constructor

Reply:
👍

Reply:
:+1: